### DISCUSSION TITLE
ExecStart=/etc/init.d/globaleaks start (code=exited, status=1/FAILURE)

### DISCUSSION THREAD

**Discussion by erwindegrave** [globaleaks/globaleaks-whistleblowing-software#3385] [Q&A]
Hi,

My globaleaks is not working.

When I look for a service globaleaks status, I get this:

● globaleaks.service - LSB: Start the GlobaLeaks server.
     Loaded: loaded (/etc/init.d/globaleaks; generated)
     Active: failed (Result: exit-code) since Fri 2023-03-10 11:11:38 CET; 1min 0s ago
       Docs: man:systemd-sysv-generator(8)
    Process: 4696 ExecStart=/etc/init.d/globaleaks start (code=exited, status=1/FAILURE)
        CPU: 785ms

Mar 10 11:11:38 globaleaks systemd[1]: Starting LSB: Start the GlobaLeaks server....
Mar 10 11:11:38 globaleaks globaleaks[4696]: Starting GlobaLeaks daemon: globaleaksStarting ...Enabling Globaleaks Network Sandboxing...done.
Mar 10 11:11:38 globaleaks globaleaks[4736]: mkdir: no se puede crear el directorio «/dev/shm/globaleaks»: El fichero ya existe
Mar 10 11:11:38 globaleaks globaleaks[4740]: Pidfile /var/globaleaks/globaleaks.pid contains non-numeric value
Mar 10 11:11:38 globaleaks globaleaks[4696]: failed.
Mar 10 11:11:38 globaleaks systemd[1]: globaleaks.service: Control process exited, code=exited, status=1/FAILURE
Mar 10 11:11:38 globaleaks systemd[1]: globaleaks.service: Failed with result 'exit-code'.
Mar 10 11:11:38 globaleaks systemd[1]: Failed to start LSB: Start the GlobaLeaks server..

Anyone any idea what is going on and how I can solve this?

Thanks

**Comment by erwindegrave** [comment]
Can anyone help me with this issue? @evilaliv3?

**Comment by erwindegrave** [comment]
At the end I could solve it, for anyone that maybe interested.
I deleted the globaleaks.pid in /var/globaleaks/.
Then I did, as root, a restart: systemctl restart globaleaks

And it is up and running again.

**Comment by evilaliv3** [comment]
Thank you @erwindegrave for reporting this and i'm sorry for the late reply but i'm happy you solve the issue.

Actually the PId file is always managed automatically and should be always correct but probably due to a specific bug in which you incurred it was created incorrectly.
If you still have the file would be interesting to look at it to try to understand the bug and implement an automatic resolution of the condition.

**Comment by erwindegrave** [comment]
@evilaliv3 I have deleted the file from the server without making a copy. I do have a backup from November 2022, and there the globaleaks.pid file is empty.
Sorry I cannot help you further.

**Comment by erwindegrave** [comment]
Hi @evilaliv3 
I had the same issue again, and to make it work again, I had to delete the file /var/globaleaks/globaleaks.pid, and do a restart: systemctl restart globaleaks
[globaleaks.pid.zip](https://github.com/globaleaks/GlobaLeaks/files/11835692/globaleaks.pid.zip)

And this time I made a copy of the globaleaks.pid file, which I have attached here in zip format (it did not allow me to upload the pid file directly).

[globaleaks.pid.zip](https://github.com/globaleaks/GlobaLeaks/files/11835692/globaleaks.pid.zip)
